Sources/Realtime/Channel.swift

```diff
@@ -93,12 +93,31 @@ public class Channel {
   /// Refs of stateChange hooks
   var stateChangeRefs: [String]
 
+  /// Initialize a Channel
+  /// - parameter topic: Topic of the Channel
+  /// - parameter options: Optional. Options to configure channel broadcast and presence
+  /// - parameter socket: Socket that the channel is a part of
+  public convenience init(topic: ChannelTopic, options: ChannelOptions = ChannelOptions(), socket: RealtimeClient) {
```

Comment:
I don't think this should be `public`, initializing channel will be executed by the client only.

Reply:
Will change!